### PR TITLE
implement basic injector and hook

### PR DESCRIPTION
**Pull request by muratcanakcay**

### CODE REVIEW

File: src/Hooks/dllmain.cpp [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#include <Windows.h>
```
Comment (ccpp-apriorit) [inline src/Hooks/dllmain.cpp]: Use precompiled header pch.h for external includes

File: src/Injector/inject.cpp [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+int inject(DWORD PID)
+{
+    // path to dll
+    //LPCSTR DllPath = "../../Hooks/debug/Hooks.dll";
+    LPCSTR DllPath = "C:\\CodeBase\\Apriorit-Project\\build-src-Desktop_Qt_6_3_1_MinGW_64_bit-Debug\\Hooks\\debug\\Hooks.dll";
+
+
+    // Open a handle to target process
+    HANDLE hProcess = OpenProcess(PROCESS_ALL_ACCESS, FALSE, PID);
```
Comment (ccpp-apriorit) [inline src/Injector/inject.cpp:9]: Check the result of this and next functions

File: src/Injector/inject.cpp [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <Windows.h>
+
+int inject(DWORD PID)
+{
+    // path to dll
+    //LPCSTR DllPath = "../../Hooks/debug/Hooks.dll";
+    LPCSTR DllPath = "C:\\CodeBase\\Apriorit-Project\\build-src-Desktop_Qt_6_3_1_MinGW_64_bit-Debug\\Hooks\\debug\\Hooks.dll";
```
Comment (ccpp-apriorit) [inline src/Injector/inject.cpp]: I understood what you meant when told about absolute path. There should be no hardcode at all. This path should be passed to the function somehow

File: src/Hooks/dllmain.cpp (hunk 1 above)
Comment (ccpp-apriorit) [reply src/Hooks/dllmain.cpp]: You can look how Mateush done this

Comment (muratcanakcay) [reply src/Hooks/dllmain.cpp]: done

File: src/Injector/inject.cpp (hunk 3 above)
Comment (muratcanakcay) [reply src/Injector/inject.cpp]: done.

File: src/Injector/inject.cpp (hunk 2 above)
Comment (muratcanakcay) [reply src/Injector/inject.cpp:9]: done

**Review by ccpp-apriorit — APPROVED** [review]

**Pull request merged**